testing/deployer/sprawl/internal/tfgen/templates/container-app-dataplane.tf.tmpl

```diff
@@ -18,9 +18,24 @@ resource "docker_container" "{{.Node.DockerName}}-{{.Service.ID.TFString}}-sidec
   }
 
   env = [
-{{- range .Env }}
```

Comment:
I changed all these templates to push more logic into the templates themselves. Personally I'd bet we could do this all in Go instead: Generate a typed HCL tree and write it out, instead of using error-prone text templating

Reply:
Yep, I was thinking tf modules would be a nice middle ground. That said, I'm still not convinced this TF approach is a net win over `testcontainers`.